packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -267,6 +276,14 @@ const MediaControlsBase = kind({
 		 */
 		playIcon: PropTypes.string,
 
+		/**
+		 * Disables the media "play"/"pause" button.
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		playPauseButtonDisabled: PropTypes.bool,
```

Comment:
Since this disables handling the key event for play/pause as well, what do you think of renaming to `playPauseDisabled`?

Reply:
Nah, we can't really do that at this point, sadly. We could deprecate them but let's stick with what you have. Thanks for pointing that out!